packages/cli/src/commands/db/revert.ts

```diff
@@ -33,7 +33,10 @@ export async function main(
 	}
 
 	if (!lastMigration.down) {
-		logger.error('The last migration was irreversible and cannot be reverted.');
+		const message = lastMigration.name
```

Comment:
Why is it possible to have nameless migrations? Can we please enforce having migration names?

Reply:
I think this might happen even if we enforce the names.

Say you upgraded to a version and encountered an issue. Then you revert n8n version BEFORE running the command. In this case, the migration file (the actual implementation of `ReversibleMigration` does not exist), you might encounter this issue.

Reply:
I can take a look, but changing the interface and making sure that all cases are accounted for seems beyond what this PR tries to achieve.

Reply:
The interface is defined in TypeORM:
 
https://github.com/n8n-io/typeorm/blob/9006e120c604bd54aeee17378f9265e4a2ea7208/src/migration/MigrationInterface.ts#L10

I think effort  to refactor that, releasing a new version, then updating this PR is not justified just to get rid of a ternary. Let me know if you disagree.

Reply:
@ivov all our migrations are actually nameless:


This migration is nameless:

```ts
class CreateProject1714133768519 implements ReversibleMigration {
  async up () {}
}
```

These migration have names:

```ts
class CreateProject1714133768519 implements ReversibleMigration {
	name = "CreateProject"

  async up () {}
}

class CreateProject1714133768519 implements ReversibleMigration {
	constructor () {
		this.name = "CreateProject"
	}

  async up () {}
}
```

If a migration has no name, then it defaults to the class name.

I haven't found a single migration which has a name property set.

I fixed this now by using `migration.name ?? migration.prototype.name`, so this should work like TypeORM does.